### PR TITLE
Add egenix-mxodbc-zopeda add-on for Plone

### PR DESCRIPTION
**Pull request by malemburg**
eGenix mxODBC Zope DA is a Zope/Plone database adapter to connect Plone to relational databases and work with them via Zope Z SQL Methods.

### CODE REVIEW

**Comment by malemburg** [comment]
Hmm, the repo doesn't have a CHANGES.txt file.

File: add-ons.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
 <p>Two different ways to integrate Solr, the Lucene based search engine, into Plone</p>
 <h3><a href="https://pypi.python.org/pypi/eea.facetednavigation">eea.facetednavigation</a></h3>
 <p>Turn folders and collections into faceted searches</p>
+<h3><a href="https://pypi.python.org/pypi/egenix-mxodbc-zopeda/">egenix-mxodbc-zopeda</a></h3>
+<p>Connect Plone to relational databases</p>
```
Comment (jensens) [inline add-ons.html:21]: Given that we have also RelStorage with a complete different scope/goal: the description could be more specific.

Comment (malemburg) [reply add-ons.html:21]: Thanks, I'll try to come up with a better short description.

What about the change log entry ? Should I create a CHANGES.txt file for this ?

**Comment by jensens** [comment]
as the Paragon initiative died silently I close this PR

**Pull request closed without merging**